pkg/image/prune/prune_test.go

```diff
@@ -870,6 +871,28 @@ func TestImagePruning(t *testing.T) {
 			expectedImageDeletions: []string{},
 			expectedStreamUpdates:  []string{},
 		},
+		"image exceeding limits with namespace specified": {
+			pruneOverSizeLimit: newBool(true),
+			namespace:          "foo",
+			images: imageList(
+				unmanagedImage("id", "otherregistry/foo/bar@id", false, "", ""),
+				sizedImage("id2", registryURL+"/foo/bar@id2", 100, nil),
+				sizedImage("id3", registryURL+"/foo/bar@id3", 200, nil),
+			),
+			streams: streamList(
+				stream(registryURL, "foo", "bar", tags(
```

Comment:
One more test having at least two image streams where one belongs to different namespace?